ruby/loops.md

```diff
@@ -0,0 +1,38 @@
+# Loops
+Now that we have gotten our hands a little dirty with ruby, let's discuss the
+concept of loops. Loops allow a program to execute particular tasks as
+many times as needed to achieve a particular goal. This is important because
+thousands of steps within a program can be completed within seconds, increasing
```

Comment:
I definitely agree and initially thought I need to add an example but went against my better judgment.